src/index.js

```diff
@@ -0,0 +1,127 @@
+import { isComp, checkBox } from './script';
+import {
+  listUpdate, addToDo, clearAll, clearAllComp, itemDelete, capitalize,
+} from './addremove';
+import './style.css';
+
+let dataList = [];
+
+const index = (dataList) => {
+  for (let i = 0; i < dataList.length; i++) { /* eslint-disable-line no-plusplus */
+    dataList[i].index = i + i;
+  }
```

Comment:
- Instead of disabling the Eslint rule. Please try this code
` for (let i = 0; i < dataList.length; i+1) { dataList[i].index = i + i; }`

Reply:
Thank you for the review 👍